.gitlab-ci.yml

```diff
@@ -0,0 +1,23 @@
+variables:
+  DOCKER_HOST: tcp://docker:2375
+
+services:
+  - docker:stable-dind
+
+stages:
+  - build
+  - test
+
+include:
+  - remote: "https://github.com/tnir/trivy/raw/gitlab-ci-template/contrib/Trivy.gitlab-ci.yml"
```

Comment:
After https://github.com/aquasecurity/trivy/pull/376 got merged, this line will be replaced with `https://github.com/aquasecurity/trivy/raw/gitlab-ci-template/contrib/Trivy.gitlab-ci.yml`.

Reply:
This line was replaced with `https://github.com/aquasecurity/trivy/raw/`**master**`/contrib/Trivy.gitlab-ci.yml`